src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleExtensionMetadata.java

```diff
@@ -354,8 +354,7 @@ private static Optional<String> makeUseRepoCommand(
       commandParts.add(extensionName);
     }
     commandParts.addAll(repos);
-    return Optional.of(
-        String.format("buildozer '%s' //MODULE.bazel:all", String.join(" ", commandParts)));
+    return Optional.of(String.format("'%s'", String.join(" ", commandParts)));
```

Comment:
nit -- maybe reuse the "Collectors.joining" idiom above?

Reply:
Done